src/new.rs

```diff
@@ -133,10 +117,10 @@ fn create_file<P: AsRef<Path>>(name: P, content: &[u8]) -> Result<()> {
 
     let mut file = try!(OpenOptions::new()
         .write(true)
-        .create(true)
+        .create_new(true)
```

Comment:
Yep, this reports an error if the file already exists, otherwise works fine. Apparently this is the preferred way to check if a file exists since it prevents race conditions.

I think the error is clear enough - here's the output:

```
$ cobalt new

[info]   Using config file .cobalt.yml
[error]  File exists (os error 17)
[error]  Could not create post
```